.github/CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ commands are available:
    considered for merge.
 1. Garden
    [maintainers](https://github.com/orgs/zendeskgarden/teams/maintainers)
-   will manage the squashed merge to master, using your PR title and
+   will manage the squashed merge to main, using your PR title and
```

Comment:
Please use "the main branch" in this context. See backing issue for details.

Reply:
Please also update the Code of Conduct. See backing issue for details.

Reply:
Haha who reads the issues, right? 😄 Updated